dask/array/tests/test_array_core.py

```diff
@@ -1435,6 +1465,23 @@ def test_store_locks():
         if i == 9:
             assert False
 
+    # Verify number of lock calls
```

Comment:
Can you expand this comment slightly to say why it should be `2 * nchunks` in one case and `nchunks` in the other? I know why right now, but future maintainers may not.

Reply:
Sure. Though I'd prefer to put that comment near the branch below if that is ok.

Reply:
Besides this comment, all other things I pointed out were nits that can be ignored (up to you).

Reply:
Sure. Was working my was through them. Just now got this one.